about.html

```diff
@@ -2,7 +2,7 @@
 layout: index
 ---
 <p>
-  Fireslime is a small indie team originally from Brazil working their way into the gamedev. We are also proud supporters of the Flame Engine community and ecosystem.
+Fireslime is a small indie team originally from Brazil working their way into the gamedev. We are also proud supporters of the <a href="https://flame-engine.org" />Flame Engine</a> community and ecosystem.
```

Comment:
I am not sure if supporters is the best choice of words here, maybe maintainers or something to that effect?